File: yangxj96-serve/yangxj96-serve-system/src/test/java/io/github/yangxj96/server/system/ApplicationTest.java
package io.github.yangxj96.server.system;

import io.github.yangxj96.common.utils.AesUtil;
import lombok.extern.slf4j.Slf4j;
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;

@Slf4j
@SpringBootTest
class ApplicationTest {


    @Test
    void aesTest() {

        var str = "hello world";

        String encrypt = AesUtil.encrypt(str);

        log.info("编码后:{}", encrypt);

        String decrypt = AesUtil.decrypt(encrypt);

        log.info("解码后:{}", decrypt);

    }

}

File: yangxj96-serve/yangxj96-serve-gateway/src/test/java/com/yangxj96/saas/server/gateway/ApplicationTest.kt
/*
 * Copyright (c) 2018 - 2023
 * 作者：杨新杰(Jack Young)
 * 邮箱：yangxj96@gmail.com
 * 博客：www.yangxj96.com
 * 日期：2023-09-13 00:04:11
 * Copyright (c) 2018 - 2023
 */

package com.yangxj96.saas.server.gateway

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import reactor.core.publisher.Mono
import java.time.Duration

@SpringBootTest
internal class ApplicationTest {

    @Test
    fun test01() {
        Mono
            .defer { Mono.just("hello world") }
            .delayElement(Duration.ofMillis(5))
            .subscribe()
        Assertions.assertTrue(true)
    }

}